lib/internal/bootstrap/loaders.js

```diff
@@ -126,6 +130,9 @@ const runtimeDeprecatedList = new SafeSet([
           'DeprecationWarning',
           'DEP0111');
       }
+      if (legacyWrapperList.has(module)) {
+        return nativeModuleRequire('internal/legacy/processbinding')[module]();
+      }
```

Comment:
Note that due to the way this is currently implemented, `process.binding("util")` will now return a new object every time it’s called.

---

I’m adding wrapper caching in <https://github.com/nodejs/node/pull/38132>.